packages/plugins/script/Sample.s.sol

```diff
@@ -11,7 +11,7 @@ contract Sample is Sphinx {
     MyContract1 myContract;
 
     function configureSphinx() public override {
-        sphinxConfig.projectName = "my_sphinx";
+        sphinxConfig.projectName = "Simple_Project";
```

Comment:
This is a rider. It just makes the sample file work immediately with the `sphinx.lock` file in the plugins package.